src/styles.scss

```diff
@@ -1467,6 +1467,25 @@ hr {
   margin-top: 1.5rem;
 }
 
+// Icon colors used by DOI/ORCID exporter
+.progress-icon.primary {
+  color: mat-color($dockstore-app-primary);
```

Comment:
seems like something angular covers already using the color directive (i.e. `color="primary"`)